integration-tests/codestarts/src/test/resources/__snapshots__/VaadinExtensionCodestartTest/testApplicationContents/src_main_java_ilove_quark_us_example_MainView.java

```diff
@@ -0,0 +1,46 @@
+package ilove.quark.us.example;
```

Comment:
good package name :)

Reply:
It's generated by Quarkus, not my idea. But, yeah, nice one :)

Reply:
hehe, I had this feeling :D but still, I smiled at it. thanks!